ansible/inventory.yml

```diff
@@ -45,7 +45,10 @@ hosts:
         rhel7-s390x-1: {ip: 148.100.86.101, user: linux1}
 
     - iinthecloud:
-        ibmi72-ppc64_be-1: {ip: 65.183.160.62, user: nodejs}
+        ibmi73-ppc64_be-1: {ip: 65.183.160.62, user: nodejs}
+
+    - rzkh:
+        ibmi73-ppc64_be-4: {ip: 185.113.4.148, user: nodejs}
```

Comment:
```suggestion
```

The new host should either be test (in the section below) or release (as is here) but not both.